### PR TITLE
Iss hipangular 25 broken translations

### PR DESCRIPTION
**Pull request by PoornimaKaruppasamy**
Check broken translations

### CODE REVIEW

File: app/notifications/notifications.component.ts [hunk 1]
```diff
@@ -27,8 +31,18 @@ export class NotificationsComponent implements OnInit {
...
         (error: any) => {
           this.notificationsResponseHandled = true;
-          this.toasterService.pop( 'error', 'Error', 'Not able to fetch your notifications.');
+          this.getTranslatedData = this.translatedData('Not able to fetch your notifications');
+          this.toasterService.pop('error', 'Error', this.getTranslatedData);
         }
       );
   }
+
+  translatedData(data: any) {
```
Comment (patrickrobrecht) [inline app/notifications/notifications.component.ts]: The method name should be something like `getTranslatedString`.

File: app/notifications/notifications.component.ts [hunk 2]
```diff
@@ -27,8 +31,18 @@ export class NotificationsComponent implements OnInit {
       .catch(
         (error: any) => {
           this.notificationsResponseHandled = true;
-          this.toasterService.pop( 'error', 'Error', 'Not able to fetch your notifications.');
+          this.getTranslatedData = this.translatedData('Not able to fetch your notifications');
```
Comment (patrickrobrecht) [inline app/notifications/notifications.component.ts]: Having a local variable for the translated string should be enough, no declaration in the class needed.

File: i18n/de.json [hunk 3]
```diff
@@ -22,13 +22,19 @@
...
   "creation time": "Erstelldatum",
   "topic details": "Details",
-  "no subtopic": "Für diesen Inhalt existieren keine Unterthemen",
+  "no subtopic": "Für diesen Inhalt existieren keine Unterthemen.",
+  "parent topics": "Übergeordneten Themen",
+  "Topic updated" : "Thema aktualisiert",
+  "Status updated": "Status aktualisiert",
 
   "email placeholder": "name@provider.de",
   "first name": "Vorname",
```
Comment (patrickrobrecht) [inline i18n/de.json]: Please use `first name`, `last name`, `role` in all occurences - then we'll just need one translation.

**Comment by maltehol** [comment]
Why did you revert Patricks changes from develop? ( https://github.com/HiP-App/HiP-CmsAngularApp/pull/99/commits/4a7cf9a98c8008ec4136ed75f01beddbbbe1d133 )

**Comment by PoornimaKaruppasamy** [comment]
Just now I checked broken style in User-list component. I reverted back the commit.

**Pull request merged**